packages/suite-desktop-core/e2e/tests/dashboard/assets.test.ts

```diff
@@ -0,0 +1,46 @@
+import { test, expect } from '../../support/fixtures';
+
+test.describe('Assets', { tag: ['@group=suite'] }, () => {
+    test.use({
+        emulatorStartConf: { wipe: true },
+        emulatorSetupConf: { needs_backup: true },
+    });
+
+    test.beforeEach(async ({ onboardingPage }) => {
+        await onboardingPage.completeOnboarding();
+    });
+
+    test('User can initiate buy from Assets in table view', async ({ assetsPage, marketPage }) => {
+        await assetsPage.tableIcon.click();
+        await assetsPage.buyAssetButton('btc').click();
+        expect(marketPage.section).toBeVisible();
+    });
+
+    test('User can initiate buy from Assets in grid view', async ({ assetsPage, marketPage }) => {
+        await assetsPage.gridIcon.click();
+        await assetsPage.buyAssetButton('btc').click();
+        expect(marketPage.section).toBeVisible();
+    });
+
+    test('New asset is shown in both grid and row', async ({
```

Comment:
This test doesnt do much verification on desktop since the snapshots are disabled. I will push for getting the resolution parameter for electron ASAP in the new year